help/domains/bio/bioinformatics.md

```diff
@@ -0,0 +1,406 @@
+
+### Natural biopolymers in FASTA-like notation
+
+Peptide sequences are displayed with color highlighting based on amino acid properties.
+DNA sequences are colored to represent different nucleotides.
+
+![FASTA peptides rendering](img/FASTA_peptides_render.png)
+
+### Natural and non-natural polymers in HELM notation
+
+HELM structures are displayed with colors corresponding to each monomer.
+If you have loaded a monomer library for HELM structures
+(you can do this by opening the column properties,
+accessed by left-clicking on the column header, and selecting "Manage libraries"),
+you gain access to previewing macromolecules
+with full support of circular and branching structures.
+
+![HELM peptides rendering](img/HELM_render.png "HELM peptides rendering")
+
+### 3D structures
+
+For PDB structures, Datagrok generates graphical previews directly within the dataframe cells.
+You can click on a cell to open a separate 3D viewer,
+which allows you to rotate, zoom, and change the color scheme.
+
+### Trees
+
+[Newick trees](https://en.wikipedia.org/wiki/Newick_format) are parsed
+and represented as a dataframe containing nodes and distances.
+
+<!--- TODO:
+This part of functionality was describes a while ago and haven't been tested by me, 
+so for now it's hidden
+
+### Sequence alignments
+When a file containing aligned sequences is imported, 
+Datagrok splits the aligned data into an alignment table by MSA
+positions (see the illustration below) 
+and performs composition analysis in a barchart on the top of this table. 
+It visualizes multiple sequence alignments with long monomer identifiers.
+
+The composition barchart is interactive, 
+the corresponding rows could be selected by clicking on the segment. 
+The rows are also highlighted in other open visualizations (such as scatter plots) 
+when you hover over the bar. 
+This enables interactive data exploration, 
+including on-the-fly statistical analysis of differences in measured values (activity)
+associated with sequences.
+
+For identifiers that do not fit in a cell, an ellipsis is shown.
+
+![Visualize Sequence Alignments](../../uploads/macromolecules/macromolecules-6.png "Visualize Sequence Alignments")
+
+Sequence alignment visualization allows for manually modifying the alignment in the case when automatic alignment result
+is not satisfactory. The composition analysis plot changes interactively, providing a quick indication of how good the
+alignment is.
+
+![Manual alignment by editing sequence in free text](../../uploads/macromolecules/macromolecules-1.gif "Manual alignment by editing sequence in free text")
+
+-->
+
+## Data editing
+
+Datagrok offers custom data editing capabilities directly within the dataframe for certain data types:
+
+* For DNA, RNA, and protein sequences in FASTA-like format, you can edit the sequences as needed.
+* If you have loaded the HELM monomer library, you can edit HELM sequences.
```

Comment:
We should specify the way how to load monomer library - possibly another page with monomer libraries description